lib/stm32/common/rtc_common_l1f024.c

```diff
@@ -335,4 +340,64 @@ void rtc_time_set_time(uint8_t hour, uint8_t minute, uint8_t second, bool use_am
 	rtc_time_set_minute(minute);
 	rtc_time_set_second(second);
 }
-/**@}*/
+
+/*---------------------------------------------------------------------------*/
+/** @brief Gets the RTC DEC time hour value
+
+@details Requires unlocking the RTC write-protection (RTC_WPR)
+*/
+uint8_t rtc_time_get_hour(void)
+{
+    return _rtc_bcd_to_dec((RTC_TSTR >> RTC_TSTR_HU_SHIFT) & (RTC_TSTR_HU_MASK | (RTC_TSTR_HT_MASK << 4)));
+}
+
+/*---------------------------------------------------------------------------*/
+/** @brief Gets the RTC DEC time minute value
+
+@details Requires unlocking the RTC write-protection (RTC_WPR)
+*/
+uint8_t rtc_time_get_minute(void)
+{
+    return _rtc_bcd_to_dec((RTC_TSTR >> RTC_TSTR_MNU_SHIFT) & (RTC_TSTR_MNU_MASK | (RTC_TSTR_MNT_MASK << 4)));
```

Comment:
Did you actually test any of this?  TSTR is the timestamp register, not the time register. Those are _not_ the same thing!